reindexer/trigger_reindex.py

```diff
@@ -149,30 +184,10 @@ def create_shards(client, prefix, desired_version, count, table_name):
         table_name=table_name
     )
 
-    # Trigger an immediate capacity increase on the DynamoDB table.
-    #
-    # If we just start the reindexer, the autoscaling will eventually warm up
-    # the table, but only after hitting lots of ProvisionedThroughputExceeded
-    # exceptions.  This should give a smoother start to the reindexer.
-    #
-    # Note: this does not disable the autoscaling rules, and if you run this
-    # without starting the reindexer, it eventually scales back down.
-    #
-    client.update_table(
-        TableName='SourceData',
-        ProvisionedThroughput={
-            'WriteCapacityUnits': 50,
-            'ReadCapacityUnits': 5
-        },
-        GlobalSecondaryIndexUpdates=[
-            {
-                'Update': {
-                    'IndexName': 'reindexTracker',
-                    'ProvisionedThroughput': {
-                        'WriteCapacityUnits': 50,
-                        'ReadCapacityUnits': 5
-                    }
-                }
-            }
-        ]
-    )
```

Comment:
I wonder if we will need this, or something like it again?

Reply:
Maybe. But I don’t think it’d be hard to add back.

I removed it because it hits errors on the Sierra reindexes – by the time we reach this line, table capacity has already scaled past 50 write units, and it errors because it thinks you’re trying to decrease capacity.

Taking it out and taking a small hit on Miro reindexes seemed easier than adding error handling logic for now.